README.md

```diff
@@ -12,15 +33,21 @@ A minimalist library for Roman numeral operations.
 - Get Roman numerals within a range 📡
 
 ## 📦 Installation
-
-It can be installed with npm.
+To get started with ToRoman,
+ you need NPM already installed or you can do
+```sh
```

Comment:
`npm install` to install `npm`? 💀

Reply:
Yes UX making it beginner friendly, incase someone wants to use and doesn't have NPM